firefox-ios/Client/Frontend/Autofill/Address/AddressAutofillSettingsViewController.swift

```diff
@@ -27,6 +27,9 @@ class AddressAutofillSettingsViewController: SensitiveViewController, Themeable
     /// Logger for logging messages and events.
     private let logger: Logger
 
+    let windowUUID: WindowUUID
+    var currentWindowUUID: UUID? { return windowUUID }
```

Comment:
in these cases, do we want to only expose `currentWindowUUID` and keep `windowUUID` private?

Reply:
I'm not sure that the privatization for either of these is too critical TBH. The windowUUID is almost always a constant injected during initialization,  and there's not really any big benefit (that I can see) to privatizing it (often privatizing internal values can discourage anti-patterns that arise from other controllers/code accessing the values directly in other code, but in this case for the UUID I'm not sure it really matters much).

Having said all that 😄 , if you think we should be more aggressive about making these private I'm definitely Ok with that. (I might do that in a follow-up PR though, because I think there's probably 100+ files with the UUID property like this.)

Reply:
I personally lean towards privatizing unless it is actively being used or we have a specific use case. Provides clarity that the property is only to be used within the class and cannot be set without constructor injection. Not a blocker of course, so yes follow-up PR is okay!